internal/service/pushbasedlogexport/data_source_schema.go

```diff
@@ -0,0 +1,81 @@
+package pushbasedlogexport
+
+import (
+	"context"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func DataSourceSchema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"bucket_name": schema.StringAttribute{
+				Computed:            true,
+				Description:         "The name of the bucket to which the agent will send the logs to.",
```

Comment:
i don't remember what each one was used for, do we need both?

Reply:
https://developer.hashicorp.com/terraform/plugin/framework/handling-data/schemas#markdowndescription 
We should use both as a best practice.
Quoting from TF documentation, _"MarkdownDescription is used to provide a markdown-formatted version of the description to tooling like [terraform-plugin-docs](https://github.com/hashicorp/terraform-plugin-docs)."_